index.html

```diff
@@ -1694,8 +1711,11 @@ <h3>Algorithm</h3>
               has a <a>local context</a>, set <var>active context</var> to the result
               to the result of the
               <a href="#context-processing-algorithm">Context Processing algorithm</a>,
-              passing <var>active context</var> and the value of the
-              <var>term</var>'s <a>local context</a> as <var>local context</var>.</li>
+              passing <var>active context</var>, the value of the
+              <var>term</var>'s <a>local context</a> as <var>local context</var>,
+              <span class="changed">and <var>from sealed term</var> set to the value
```

Comment:
Wow... that would mean that a "sealed type" could redefine the context of the value of a "sealed property", even if the type and the property are sealed by different contexts?

```json
{
  "@context": [
    {
      "@sealed": true,
      "a_prop1": "http://example.org/a#prop1"
    },
    {
      "@sealed": true,
      "b_class1": {
        "@id": "http://example.org/b#class1",
        "@context": null
       }
    }    
  ],
  "a_prop1": {
      "@type": "b_class1",
      "a_prop1": "should this property be unsealed?"
  }
}
```

This is kind of counter-intuitive for me, but it's probably because I'm not familiar yet with "type-scoped contexts"...

Reply:
Indeed, but it falls out of the logic. But, @dlongley’s notion that sealing doesn’t hold under non sealed terms my alieviate this; perhaps we need to be say that sealed terms can cause their values to be unsealed, and not simply wipe out the context.